### PR TITLE
Host dash

### LINKED ISSUE #60: Host Dashboard - Create Yard
As a host,
When I visit my dashboard
I see a button to create a yard

### PR DESCRIPTION
**Pull request by Gvieve**
### Add
* Host Dashboard route, controller, action and feature tests to create a yard
* Host Yards new form with basic information
__WIP: checkbox collection for purposes table, need to get data contract from BE__

Close #60

### CODE REVIEW

File: app/views/shared/_nav_bar.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<section class='nav-bar'>
+  <h3>Welcome <%= current_user %>></h3>
+  <%= button_to 'Host Dashboard', host_dashboard_index_path, method: :get %>
+  <%= button_to 'Renter Dashboard', renter_dashboard_index_path, method: :get %>
+  <%= button_to 'Logout', logout_path, method: :delete %>
+</section>
```
Comment (domo2192) [inline app/views/shared/_nav_bar.html.erb:6]: I like the nav bar and the idea of our user being able to shift between host and renter dashboard.

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+    resources :dashboard, only: :index
+    resources :yards, only: [:new, :create]
+  end
+
+  namespace :renter do
+    resources :dashboard, only: :index
+  end
+
+  delete "/logout", to: "sessions#destroy"
 end
```
Comment (domo2192) [inline config/routes.rb:18]: I like this route setup. Which route dashboard do we want a user to go into when they login? That what our route will have to be refactored into.

File: app/views/shared/_nav_bar.html.erb (hunk 1 above)
Comment (dev-named-jordan) [reply app/views/shared/_nav_bar.html.erb:6]: Did we also say we wanted to put the search page in the nav bar as well, or no?

Comment (Gvieve) [reply app/views/shared/_nav_bar.html.erb:6]: Thanks! @jordanfbeck0528 you are correct, we planned to add the Search Page in the navbar. The reason we didn't include it yet is we want to chat with the group as far as where that page will live route wise. We are thinking within the renter dashboard/yards/search but want to be sure everyone agrees.

File: config/routes.rb (hunk 2 above)
Comment (Gvieve) [reply config/routes.rb:18]: We were thinking send them to the renter dashboard. @domo2192 does that sound good to you gents?

**Review by dev-named-jordan — APPROVED** [review]
Looks great! I think we can merge this once we take care of the tiny merge conflict in routes?

**Review by jbranham1 — APPROVED** [review]
Looks great!

**Pull request merged**